pkg/tracker/runner.go

```diff
@@ -102,9 +105,10 @@ func (r *Runner) Start(ctx context.Context, exitCh chan int) error {
 					//runnerLog.Info("Running trackers...")
```

Comment:
all of these commented logs should be debug logs, but lets leave for another PR.